todo_app/templates/index.html

```diff
@@ -7,11 +7,20 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+  <h2>Add item</h2>
+  <form method="POST" action="add">
+    <label for="title">Title:</label>
+    <input id="title" type="text" name="title"/>
```

Comment:
minor but might be slightly nicer to have a `required` attribute here, to avoid creating empty tasks